docs/performance/loadtest.md

```diff
@@ -0,0 +1,102 @@
+# Load Testing
+
+This document describes how to perform the load testing in Vald cluster or Vald agent using Vald Load Testing Tools.
+
+## Overview
+
+We develop our own Load Testing Tools to perform the load testing in Vald cluster or Vald agent. You can perform the load test on Vald to test the insert and search performance in Vald.
+
+## Prerequisite
+
+- Vald Cluster or Vald Agent
+
+    Please refer to [here](https://vald.vdaas.org/docs/tutorial/get-started/) for the installation guide of Vald Cluster or Vald Agent.
+
+- Docker
```

Comment:
Could you show the minimum version?

Reply:
Sorry I am not sure the minimum version of docker to execute the load testing tools... 
I think it works on most version of docker though.... 
What do you think?

Reply:
I think you show the version of docker which you run the load test.